src/Illuminate/Routing/RoutingServiceProvider.php

```diff
@@ -137,9 +138,12 @@ protected function registerPsrRequest()
                 return (new PsrHttpFactory($psr17Factory, $psr17Factory, $psr17Factory, $psr17Factory))
                     ->createRequest($app->make('request'));
             }
+
             if (class_exists(DiactorosFactory::class)) {
                 return (new DiactorosFactory)->createRequest($app->make('request'));
             }
+
+            throw new Exception('Unable to resolve PSR request. Please install nyholm/psr7 or laminas/laminas-diactoros.');
```

Comment:
We should probably replace the last sentence here with `Please install symfony/psr-http-message-bridge.`

Reply:
Probably should be both since that Psr17Factory comes from the nyholm package?